paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -253,7 +253,7 @@ void CpuPassStrategy::EnableMKLDNN() {
              // Disabled due to topology-dependent speed-up
              // "fc_mkldnn_pass",
              // "fc_act_mkldnn_fuse_pass",
-             "batch_norm_act_fuse_pass",
+             "batch_norm_act_fuse_pass", "elt_act_onednn_fuse_pass",
```

Comment:
@tsocha  Hi, could you please commit it again under paddle official docker?  Please try this, Ok ? Because Baidu developers are now all work under docker. Let make sure align with Baidu cause they are customers. Baoachun mentioned this line many times, only we made sure that even under Baidu official docker, it is still like this, we can proceed more converstion with them.